tablenamer/tablenamer.go

```diff
@@ -0,0 +1,115 @@
+package tablenamer
+
+import (
+	"unicode"
+
+	"github.com/samonzeweb/godb/tablenamer/plural"
+	"sync"
+)
+
+// TableNamer function type is used set table naming function to build table name from struct's name
```

Comment:
Bad type name in comment (thanks vscode and its excellent Go tools integrationt).